domains/codeidea.json

```diff
@@ -4,6 +4,6 @@
         "email": "rodrigojesse48@gmail.com"
     },
     "record": {
-        "CNAME": "allenb98.github.io"
+        "CNAME": "allenb98.github.io/2tor/"
```

Comment:
sorry for late reply, but i dont think that will work, just rename the website repo to allenb98.github.io and change back this file to the first one (aka the one i originally requested reviews)